### PR TITLE
Added javascript folder and file

### PR DESCRIPTION
**Pull request by gsuitebiz**

### CODE REVIEW

**Comment by binarymason** [comment]
@gsuitebiz congrats on making the very first PR! 👍 🎉 

Great work!

Can you refactor this to return an array of numbers and "fizz buzzes" instead of just `console.log`? If you want to see visual output, you can always create your array called `numbers` and run `console.log(numbers)`.

**Comment by gsuitebiz** [comment]
oops :) Meant to do that! Thanks!

**Comment by binarymason** [comment]
Looking good @gsuitebiz ✨ 

You mind running `eslint` on this file?  Here are the instructions to get `eslint` running: http://eslint.org/docs/user-guide/getting-started.  Once you run `eslint`, I think we may have a winner. 🏅 

👍

**Comment by gsuitebiz** [comment]
The only "error" I am getting at this point is that the function isn't
being used. Do I need to call it for this challenge? I assumed that it was
just writing the actual function that was necessary. And I am unfamiliar
with eslint but I went through the instructions, ran it on my file,
corrected spacing etc.. Is there something that I need to do besides change
the current code on git?

**Comment by binarymason** [comment]
hey @gsuitebiz, 

Yeah, eslint is a go-to tool that a lot of programmers use.  Great to have as a linter in your editor so it yells at you while you code. 😉 

We can ignore the warning that your function is not being called for now.  It's helpful in big projects to reduce "bloat" of unused code lying around, but that doesn't really apply here.

Once you get `eslint` happy with formatting go ahead and push up your changes and let me know.  I think this PR is ready to be merged as an official solution. 👍

**Comment by gsuitebiz** [comment]
Just updated file

File: fizzbuzz/javascript/gsuitebiz/fizzbuzz.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+		}else if(remainder5 == 0){
+			
+			arrayOfNumbers.push("buzz");
+		}else{
+			
+			arrayOfNumbers.push(i);
+		}
+		
+	}
+
```
Comment (binarymason) [inline fizzbuzz/javascript/gsuitebiz/fizzbuzz.js]: Oops, looks like you need to `return arrayOfNumbers` here

**Comment by binarymason** [comment]
@gsuitebiz FYI -- I am adding an `eslint` config.  See here https://github.com/charlottejuniordevs/challenges/blob/master/fizzbuzz/javascript/gsuitebiz/fizzbuzz.js for an updated version of your code after running `eslint`. 👍

**Comment by gsuitebiz** [comment]
Awesome, thanks!

**Pull request merged**